pkgs/by-name/po/posting/package.nix

```
@@ -0,0 +1,49 @@
+{ lib
+, python3
+, fetchFromGitHub
+}:
+
+python3.pkgs.buildPythonApplication rec {
+  pname = "posting";
+  version = "1.0.3";
+  pyproject = true;
+
+  src = fetchFromGitHub {
+    owner = "darrenburns";
+    repo = "posting";
+    rev = version;
+    hash = "sha256-1XLIAa1EQVYuphbMMtJHFdjjQ0s0DGr8fiom1zKCJGM=";
+  };
+
+  nativeBuildInputs = [
```

```suggestion
  build-system = [
```
Please use `build-system` instead.
https://github.com/NixOS/nixpkgs/blob/master/doc/languages-frameworks/python.section.md#buildpythonpackage-parameters-buildpythonpackage-parameters